packages/main/src/plugin/kubernetes/fake-informer.ts

```diff
@@ -0,0 +1,103 @@
+/**********************************************************************
```

Comment:
Should this class use `.spec` ? Can we import stuff from one `.spec` file to another `.spec` ?

If we cannot I think we should use `informer-test.ts` rather than `fake`

Reply:
> Should this class use `.spec` ? Can we import stuff from one `.spec` file to another `.spec` ?
> 

Sonarjs is not happy with: 
```
  0:1  error  Add some tests to this file or delete it  sonarjs/no-empty-test-file
```

Reply:
I would still rather have a basic test, than having this without the `.spec` format :/